bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/handler/DWDPollenflugBridgeHandler.java

```diff
@@ -0,0 +1,213 @@
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.dwdpollenflug.internal.DWDPollingException;
+import org.openhab.binding.dwdpollenflug.internal.config.DWDPollenflugBridgeConfiguration;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflug;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DWDPollenflugBridgeHandler} is the handler for bridge thing
+ *
+ * @author Johannes Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(DWDPollenflugBridgeHandler.class);
+
+    private static final String DWD_URL = "https://opendata.dwd.de/climate_environment/health/alerts/s31fg.json";
+
+    private DWDPollenflugBridgeConfiguration bridgeConfig = new DWDPollenflugBridgeConfiguration();
+
+    private @Nullable ScheduledFuture<?> pollingJob;
+
+    private @Nullable DWDPollenflug pollenflug;
+
+    private final Set<DWDPollenflugRegionHandler> regionListeners = ConcurrentHashMap.newKeySet();
+
+    private final HttpClient client;
+
+    private final Gson gson = new Gson();
+
+    public DWDPollenflugBridgeHandler(Bridge bridge, HttpClient client) {
+        super(bridge);
+        this.client = client;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            startPolling();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing DWD Pollenflug bridge handler");
+        bridgeConfig = getConfigAs(DWDPollenflugBridgeConfiguration.class);
+
+        if (bridgeConfig.isValid()) {
+            startPolling();
+            updateStatus(ThingStatus.ONLINE);
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "Refresh interval has to be at least 15 minutes.");
+        }
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Handler disposed.");
+        stopPolling();
+    }
+
+    private void startPolling() {
+        final ScheduledFuture<?> localPollingJob = this.pollingJob;
+        final DWDPollenflug localPollenflug = this.pollenflug;
+        if (localPollingJob == null || localPollingJob.isCancelled()) {
+            logger.debug("Start polling.");
+            pollingJob = scheduler.scheduleWithFixedDelay(this::poll, 0, bridgeConfig.refresh, TimeUnit.MINUTES);
+        } else if (pollenflug != null) {

```

Comment:
I think you should move this part down to `childHandlerInitialized`. And then notify only the recently added child handler and not all child handlers.

Reply:
Moved